src/renderers/shared/stack/reconciler/ReactInstanceType.js

```diff
@@ -34,4 +34,5 @@ export type ReactInstance = {
   _mountImage: any,
   // __DEV__
   _debugID: DebugID,
+  _warnedAboutRefsInRender: bool,
```

Comment:
<img src="https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwiGoPSelfLOAhVJKWMKHTleA0sQjRwIBw&url=http%3A%2F%2Fwww.hilariousgifs.com%2Fhomer-simpson-hiding-in-the-bushes%2F&psig=AFQjCNGRAm0qC0dAoQkuaywU5Q7vuKC3ow&ust=1472956645675316"/>